internal/sequencer/helpers.go

```diff
@@ -133,3 +145,50 @@ func privateKeyFromText(privkey string) (ed25519.PrivateKey, error) {
 	from := ed25519.NewKeyFromSeed(privKeyBytes)
 	return from, nil
 }
+
+// getAddressAsBytes converts an address string to a byte array. It will first
+// attempt to decode the address as bech32m, and if that fails, it will attempt
+// to decode the address as a hexadecimal string. If the address is not a valid
+// bech32m or hex string, an error will be returned. The boolean return value
+// indicates if the address was successfully decoded as bech32m. It will be true
+// if the address was decoded as bech32m, and false if it was decoded as hex or
+// couldn't be decoded.
+func getAddressAsBytes(address string) ([20]byte, bool, error) {
+	hrp, data, err := bech32.Decode(address)
+	if err != nil {
+		log.Debugf("Could not decode address as bech32: %v\n", err)
+		log.Debug("Attempting to decode address as hex")
+
+		address = strip0xPrefix(address)
+		bytes, err := hex.DecodeString(address)
+		if err != nil {
+			log.WithError(err).Error("Error decoding hex encoded address")
```

Comment:
What would happen if it is a bech32 address but they left off a character or somehow messed up the address when copying it? I think it would error out here and the error message could be misleading. Maybe we update line 159-160 to print at warning level?